src/test/mocks/@polkadot/ui-keyring/index.ts

```diff
@@ -3,9 +3,9 @@ import '@testing-library/jest-dom';
 const mockLoadAll = jest.fn();
 
 jest.mock('@polkadot/ui-keyring', () => ({
-    keyring: {
-        loadAll: mockLoadAll
-    }
+  keyring: {
```

Comment:
Could you check if you are running a different prettier config?

Reply:
I think I'm using the configured one in our project.
This fix is supposed to be included in 2.17.0. Could you please approve if it looks good other than that? Let me have a look at it after that.